addons/l10n_in_sale/models/payment_transaction.py

```diff
@@ -0,0 +1,20 @@
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class PaymentTransaction(models.Model):
+    _inherit = 'payment.transaction'
+
+    @api.model_create_multi
+    def create(self, values_list):
+        transaction = super().create(values_list)
```

Comment:
Since it's a "create multi",
```suggestion
        transactions = super().create(values_list)
```
and `for tx in transactions`